frontend/appflowy_flutter/lib/workspace/presentation/settings/pages/settings_shortcuts_view.dart

```diff
@@ -460,6 +477,64 @@ class _ShortcutSettingTileState extends State<ShortcutSettingTile> {
   }
 }
 
+class EditShortcutBtn extends StatelessWidget {
```

Comment:
For widgets used only in this file, we can make them private.
```suggestion
class _EditShortcutButton extends StatelessWidget {
```